rfcs/text/0053-new-device-fields.md

```diff
@@ -0,0 +1,127 @@
+# 0053: Additional fields for Device enrichment
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+This RFC proposes the addition of new fields to the existing `device` field set to provide more comprehensive device identification and classification capabilities. The proposed fields include vendor information, product details and device type classification to enable better device tracking, inventory management, and security monitoring. These fields will enhance the ability to uniquely identify and categorize devices across various use cases including endpoint security, asset management, and compliance monitoring.
+
+<!--
+Stage 1: If the changes include field additions or modifications, please create a folder titled as the RFC number under rfcs/text/. This will be where proposed schema changes as standalone YAML files or extended example mappings and larger source documents will go as the RFC is iterated upon.
+-->
+
+
+## Fields
+
+<!--
+Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
+-->
+
+Field | Type | Description
+-- | -- | --
+device.vendor.name | keyword | Vendor name of the device. Example: "SanDisk"
+device.vendor.id | keyword | VendorID of the device. Example: "0x1234". See [1][2]
+device.product.name | keyword | Product name of the device. Example: "Extreme V2 SSD"
+device.product.id | keyword | ProductID Example: "0xABCD".  See [1][2]
+device.type | keyword | Device type classification. Example: "Storage Device", "HID", "Audio". Ideally one of the USB defined class codes. See [3]
```

Comment:
do we need here also class ID according to the link? `07h` etc
I am not sure how unique it should be and if it adds any value.